app/views/layouts/application.html.erb

```diff
@@ -18,10 +18,11 @@
   <%= render "partials/footer" %>
 
   <%= stylesheet_link_tag "https://cdnjs.cloudflare.com/ajax/libs/sweetalert/1.1.3/sweetalert.min.css" %>
-  
+
   <%= javascript_include_tag "https://cdnjs.cloudflare.com/ajax/libs/sweetalert/1.1.3/sweetalert.min.js"  %>
   <%= stylesheet_link_tag "pickDate", "materialize.clockpicker" %>
   <%= javascript_include_tag "https://cdn.firebase.com/js/client/2.3.2/firebase.js" if current_page?(quiz_path) %>
   <%= javascript_include_tag "application", "data-turbolinks-track" => true %>
+  <div id="google-maps-scripts"></div>
```

Comment:
`application.html.erb` is the base template. Having the `div` in the base template is not a good idea. You could try adding it in line 6 of `search_with_map.htm.erb`